### DISCUSSION TITLE
Hello, How to show/format the linux top command output, so it can shown in prometheus format? Thanks.

### DISCUSSION THREAD

**Discussion by georges-code** [prometheus/prometheus#12246] [Q&A]
### Proposal

Hello, How to show/format the linux top command output, so it can shown in prometheus format? Thanks.

**Comment by roidelapluie** [comment]
I would suggest checking out https://github.com/ncabatoff/process-exporter. However, it's important to keep in mind that the large number of metrics could lead to potential cardinality issues.
